meinberlin/apps/kiezradar/signal.py

```diff
@@ -0,0 +1,4 @@
+from django.dispatch import Signal
+
+# Define custom signal with topics as an argument
+add_kiezradars_to_profile = Signal(providing_args=["instance", "topics"])
```

Comment:
the `providing_args` argument was removed in django 4.0 :)

Reply:
oh shoot